pkg/agent/openflow/pipeline.go

```diff
@@ -142,9 +167,27 @@ var (
 	// snatMarkRange takes the 17th bit of register marksReg to indicate if the packet needs to be SNATed with Node's IP
 	// or not. Its value is 0x1 if yes.
 	snatMarkRange = binding.Range{17, 17}
+	// hairpinMarkRange takes the 18th bit of register marksReg to indicate
+	// if the packet needs DNAT to virtual IP or not. Its value is 0x1 if yes.
+	hairpinMarkRange = binding.Range{18, 18}
+	// macRewriteMarkRange takes the 19th bit of register marksReg to indicate
+	// if the packet needs to be rewrite MAC address. Its value is 0x1 if yes.
```

Comment:
s/if the packet needs to be rewrite MAC address/if the packet's MAC addresses need to be rewritten

Reply:
Done.